### PR TITLE
Handle Groups and Events display

### PR DESCRIPTION
**Pull request by DaazKu**
Update the log model to display Group and Event record properly!

### CODE REVIEW

**Comment by DaazKu** [comment]
That would be best. But since we are pretty much doing the moderation over next year ¯\\_\(ツ)\_\/¯.
I think we should start collecting these kind of improvement feedback  somewhere centralized.

**Comment by linc** [comment]
* We don't want to impose closed-source name spaces on open source.
* It'll be safer to put this in Groups where the code will survive a refactor.
* It will be less work to preserve an event than refactor this entire block out later.

**Comment by DaazKu** [comment]
Makes sense. I'll do the update!

File: applications/dashboard/models/class.logmodel.php [hunk 1]
```diff
@@ -67,48 +67,42 @@ public static function endTransaction() {
...
-            case 'Event':
-                $Result =
-                    '<b>'.$this->formatKey('Name', $Data).'</b><br />'.
-                    $this->formatKey('Body', $Data).'<br />'.
-                    t('Location').': '.$this->formatKey('Location', $Data);
-                break;
-            default:
-                $Result = '';
+        $Result = '';
+        Gdn::pluginManager()->fireAs('LogModel')->fireEvent('FormatContent', [
```
Comment (linc) [inline applications/dashboard/models/class.logmodel.php]: See later in this file for how to properly use event firing in this class:

                $L = self::instance();
                $L->EventArguments['Log'] =& $LogRow;
                $L->fireEvent('BeforeInsert');

**Pull request merged**